src/models/Car.js

```diff
@@ -0,0 +1,29 @@
+class Car {
+  #name;
+  #position;
+
+  constructor(name) {
+    this.#name = name;
+    this.#position = 0;
+  }
+
+  move(randomNumber) {
+    if (this.#isMovable(randomNumber)) {
+      this.#position += 1;
+    }
+  }
+
+  #isMovable(randomNumber) {
+    return randomNumber >= 4;
+  }
+
+  getName() {
+    return this.#name;
+  }
```

Comment:
사소하지만 접근자 프로퍼티를 이용하면 public 멤버에 접근하듯 간단하게 접근할 수 있게도 가능합니다!
https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Functions/get

```suggestion
  get name() {
    return this.#name;
  }
  
  // 외부에서 car.name 으로 접근
```

Reply:
오..이런 방법도 있네요! 좋은 피드백 주셔서 감사합니다:)